engine/events/pokepic.asm

```diff
@@ -38,6 +40,43 @@ _Displaypic:
 	ld b, 1
 	jmp SafeCopyTilemapAtOnce
 
+ClearSpritesUnderPokePic:
+	ld de, wShadowOAMSprite00
+	ld hl, wShadowOAMSprite00
+	ld c, NUM_SPRITE_OAM_STRUCTS
+.loop
+	ld a, [hli]
+	cp 41
+	jr c, .next
+	cp 120
+	jr nc, .next
+	ld a, [hl]
+	cp 49
+	jr c, .next
+	cp 128
+	jr c, .clear_sprite
```

Comment:
Comments are great around these sort of "magic" numbers, even if they make sense when you're familiar with the code.
```suggestion
	; pokemon pic y is in range (40, 120)
	ld a, [hli]
	cp 41
	jr c, .next
	cp 120
	jr nc, .next
	; pokemon pic x is in range (48, 128)
	ld a, [hl]
	cp 49
	jr c, .next
	cp 128
	jr c, .clear_sprite
```
(I *think* my added comments are accurate.)